Simplenote.xcodeproj/project.pbxproj

```diff
@@ -4922,6 +4925,7 @@
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = com.codality.NotationalFlow.Intents;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "match AppStore com.codality.NotationalFlow.Intents";
```

Comment:
The best explanation I have for the changes in this diff is: Xcode needed them to make the process work.

Reply:
One thing that could be fancy would be to make this setting be:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AppStore ${PRODUCT_BUNDLE_IDENTIFIER}";
```
(And same for other instances of this for other targets and configs ofc)

---

Side note: In other projects in past company I've even gone as far as making a user-defined build setting at project level named `BASE_BUNDLE_ID = com.codality.NotationalFlow` and then at target level make each target's `PRODUCT_BUNDLE_IDENTIFIER` use `${BASE_BUNDLE_ID}` to construct its own, e.g. `${BASE_BUNDLE_ID}.Intents`… which would match (pun intended) how we do it in the `Fastfile` too.

That's probably taking it too far, especially just for this PR, and not really needed (as the pbxproj works well just the way it is, so…); just thought you'd be curious about this possibility and idea 🙂